packages/slate-editor/src/icons/ContactLayoutCard.svg

```diff
@@ -0,0 +1,7 @@
+<svg width="49" height="48" viewBox="0 0 49 48" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
I believe the icons are wrong.

They should rely on `currentColor` and `fill` CSS properties for the active state visualization:

https://github.com/prezly/slate/blob/0373c7a75633a9dcf8a0681084f79a3e2de24537/packages/slate-editor/src/components/Menu/Menu.module.scss#L129-L132

These two icons seem to have active/inactive state baked into their SVG files.

![image](https://github.com/prezly/slate/assets/370680/28bbcad9-ea44-4773-9354-cbbffd7baaad)

Reply:
Good catch. I've fixed it by modifying the SVGs to use the same color and opacity as the other similar options: a773d4f39006f7bfe214e3bcf6513a1b338794e2

That piece of CSS that you've linked is not applied in this case I think because it's a different component.